app/src/main/java/org/mozilla/focus/settings/permissions/AutoplayState.kt

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.focus.settings.permissions
+
+data class AutoplayState(
```

Comment:
This needs documentation.
Though using it like it is now seems like a different approach than of our MVI direction.
I would be curious of other approaches to this.